jwala-integration-test/src/test/postman/jwala.postman_environment.json

```diff
@@ -102,25 +102,25 @@
     },
     {
       "key": "HealthCheckGroup",
-      "value": "HEALTH CHECK 4.0",
+      "value": "CTP-GROUP-1",
       "type": "text",
       "enabled": true
     },
     {
       "key": "HealthCheckApp",
-      "value": "HEALTH-CHECK-4.0",
+      "value": "HEALTH-CHECK-4.0.1",
       "type": "text",
       "enabled": true
     },
     {
       "key": "HealthCheckJvm",
-      "value": "CTO-N9SF-LTST-HEALTH-CHECK-4.0-{{host}}-1",
+      "value": "N9SF-LTST-CTP-GROUP-1-{{host}}-1",
       "type": "text",
       "enabled": true
     },
     {
       "key": "HealthCheckWebServer",
-      "value": "CTO-N9SF-LTST-HTTP-{{host}}",
+      "value": "HTTP-{{host}}-CTP-GROUP-1",
```

Comment:
Same comment about using variables to construct names instead of using a single variable.